src/argilla/training/transformers.py

```diff
@@ -176,8 +179,15 @@ def init_pipeline(self):
                 aggregation_strategy="first",
                 device=device,
             )
+        elif self._model_class == AutoModelForQuestionAnswering:
```

Comment:
I only see `self._model_class` being initialized for `AutoModelForSequenceClassification` and `AutoModelForTokenClassification`. Where is this one set?

Reply:
In the parent class. It is a bit weirdly setup with the inheritance to the client/feedback/*